lessons/python/cpu-multiprocessing/joblib_tutorial_2_multiple_varying_args.py

```diff
@@ -0,0 +1,22 @@
+from joblib import Parallel, delayed
+from multiprocessing import cpu_count
```

Comment:
does the same, but not sure if actually using same as multiprocessing :/ i'll just import from joblib